themes/default/content/product/_index.md

```diff
@@ -12,80 +12,92 @@ subheading: |
     and tame cloud complexity -- something we call Cloud Engineering.
 
 overview:
-    title: Unite developers, infrastructure teams, and security engineers
+    title: Universal infrastructure as code for engineers 
     description: |
-        By using the industry's best languages, you can skip learning proprietary, limited domain-specific
-        languages, and enable your team to work more productively and better together.
+        Ship infrastructure at faster velocity compared to traditional tools because it supports the most popular programming languages and software engineering practices. 
 
 key_features:
     title: Key features
     items:
         - description: Build [Infrastructure as Code](/what-is/what-is-infrastructure-as-code/) in familiar languages
```

Comment:
this title should not be here. you can delete it.

Reply:
yeah, i will likely need to revisit the structure of this when i go back through and clean this up. it currently is not being rendered or used though. so you shouldnt be seeing it on the page.

Reply:
ahh nevermind. I see what you are saying. i thought you were referring to the title property.